src/main.rs

```diff
@@ -98,9 +99,24 @@ enum Command {
     },
 }
 
+#[derive(StructOpt, Debug)]
+enum Docs {
+    #[structopt(name = "build", about = "Render HTML docs locally")]
+    Build,
+
+    #[structopt(name = "format", about = "Publish HTML docs to hexdocs")]
+    Publish,
+    // #[structopt(name = "format", about = "Remove docs from hexdocs")]
+    // Revoke,
+}
+
 fn main() {
     let result = match Command::from_args() {
-        Command::Build { path, doc } => command_build(path, doc),
+        Command::Build { path } => command_build(path, false),
+
+        Command::Docs(Docs::Build) => command_build(".".to_string(), true),
+
+        Command::Docs(Docs::Publish) => command_build(".".to_string(), true).and_then(|_| todo!()),
```

Comment:
If there's a tracking issue for the docs publishing, maybe `todo! (link) `.

Reply:
This pull request is a draft and not yet ready for review. Check back later :)